exporters/metric/prometheus/prometheus_test.go

```diff
@@ -72,6 +78,11 @@ func TestPrometheusExporter(t *testing.T) {
 	expected = append(expected, `valuerecorder_count{A="B",C="D",R="V"} 4`)
 	expected = append(expected, `valuerecorder_sum{A="B",C="D",R="V"} 19.6`)
 
+	upDownCounter.Add(ctx, 10, labels...)
+	upDownCounter.Add(ctx, -3.2, labels...)
+
+	expected = append(expected, `updowncounter{A="B",C="D",R="V"} 6.8`)
```

Comment:
I'm happy with this PR, but would you add a TODO about improving the test? This doesn't actually test that the metadata changes, because the test helper strips lines of the output that start with '#'.

Reply:
Yes, the test suite doesn't really cover the change entirely, but I didn't want to start a larger refactor of the tests as part of this issue. Happy to create the TODO, and if possible, pick up the test suite improvements here in a separate PR if that's okay?

Reply:
TODO comment has been added.

Reply:
Tracking the TODO with https://github.com/open-telemetry/opentelemetry-go/issues/1320